gm4_zauber_cauldrons/data/zauber_cauldrons/functions/main.mcfunction

```diff
@@ -20,7 +20,7 @@ execute if score crystals_clock gm4_zc_data matches 3.. run advancement grant @a
 execute if score crystals_clock gm4_zc_data matches 3.. run scoreboard players set crystals_clock gm4_zc_data 0
 
 #restore broken crystals
-data merge entity @e[type=item,nbt={Item:{tag:{SkullOwner:{Id:"d15e515a-2058-459a-bfe9-ab419a0d2e7c",Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE0NzY4ODc4ODY4MjcsInByb2ZpbGVJZCI6ImQxNWU1MTVhMjA1ODQ1OWFiZmU5ZmY1NzlhMGQyZTdjIiwicHJvZmlsZU5hbWUiOiJCbHVlZmlyZTYxMCIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83MmRkMTkxZDgxYzg4NDUzYzVlM2JjMWJlMjFkYjVhMTkyZDUyNmI3MTg2YjJmNjk0ZjI4Y2JkZmNjYzYyYzNhIn19fQ=="}]}}}}},limit=1] {Item:{tag:{gm4_zauber_cauldrons:{item:"crystal",type:"resistance"},HideFlags:1,display:{Name:"\"§eCrystal of Resistance§r\""}}}}

```

Comment:
Why move the `limit=1` to after the nbt selector on this one line?

Reply:
It was bugging me that all of them had it in front of the selector except this one

Reply:
Oh I think I misunderstood the notation for what was added and what was removed XD

Reply:
How is that even hard to understand

Reply:
Honestly have no clue.